Makefile

```diff
@@ -143,34 +140,43 @@ endif
 
 DOCKER_RUN_DOCKER := $(DOCKER_FLAGS) "$(DOCKER_IMAGE)"
 
+DOCKER_BUILD_ARGS += --build-arg=GO_VERSION
+BUILD_OPTS := ${BUILD_APT_MIRROR} ${DOCKER_BUILD_ARGS} ${DOCKER_BUILD_OPTS} -f "$(DOCKERFILE)"
+ifdef USE_BUILDX
+BUILD_OPTS += $(BUILDX_BUILD_EXTRA_OPTS)
+BUILD_CMD := $(BUILDX) build
+else
+BUILD_CMD := $(DOCKER) build
+endif
+
+ifdef DOCKER_CROSSPLATFORMS
+DOCKER_CROSS = true
+endif
```

Comment:
So we have a multi-lever thing going on here.

* `DOCKER_CROSSPLATFORM`, if set, causes `DOCKER_CROSS` to be set to `true`
* `DOCKER_CROSS`, if set, causes `build_cross` to be set to `--build-arg CROSS=$(DOCKER_CROSS)` (in two distinct places).

Since no other value rather than `true` is used for `DOCKER_CROSS`, it makes sense to remove the intermediate step, and just do
```
ifdef DOCKER_CROSSPLATFORMS
BUILD_CROSS=--build-arg CROSS=true
endif

and use either directly where you're usnig `build_cross` now, or add to BUILD_OPTS.

Reply:
Updated